platforms/jvm/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/JavaCompileProblemsIntegrationTest.groovy

```diff
@@ -83,6 +83,8 @@ class JavaCompileProblemsIntegrationTest extends AbstractIntegrationSpec {
             fqid == 'compilation:java:java-compilation-error'
             details == '\';\' expected'
         }
+
+        result.error.contains("2 errors\n")
```

Comment:
❓ I assume these changes were removed previously and now they are restored. Is that correct?

Reply:
That's correct